packages/@react-spectrum/s2/stories/SegmentedControl.stories.tsx

```diff
@@ -45,7 +45,7 @@ Example.args = {
 };
 
 export const WithIcons = (args: any) => (
-  <SegmentedControl {...args}>
+  <SegmentedControl {...args} styles={style({width: '[400px]'})}>
```

Comment:
Do we still need the customWidth story since they are all customWidth now? Should the customWidth story change to rendering within a flex or grid without a customWidth, since Rob says that's the case where we won't need that.
Icon only story doesn't need a customWidth because icon's won't be truncated?

Reply:
yeah because the icons themselves are all the same width already so there isn't any issues with truncating (also it isn't text so that's another thing. not sure how an icon would truncate in the first place). 

for the other examples, some of the labels are longer than the other's causing some buttons to truncate and not others so that they all maintain the same width. 

we could update the customWidth example though and change it like you suggested